src/prefect/__init__.py

```diff
@@ -83,8 +85,15 @@
 import prefect.logging.configuration
 
 prefect.logging.configuration.setup_logging()
+
+if PREFECT_UI_URL:
+    dashboard_url = PREFECT_UI_URL.value()
+else:
+    dashboard_url = "<no dashboard available>"
+
 prefect.logging.get_logger("profiles").debug(
-    f"Using profile {prefect.context.get_settings_context().profile.name!r}"
+    f"Using profile {prefect.context.get_settings_context().profile.name!r} - View"
+    f" dashboard at {dashboard_url!r}"
 )
```

Comment:
This is the right idea but I don't think we should do this on import. Instead:

- We want to do this in `engine.py` after creating a flow run
- We'll want to include the link to the running flow run in the dashboard
- We'll want to use an INFO log level